README.md

```diff
@@ -1,2 +1,203 @@
 # root-get
-ROOT's package manager prototype
+
+- ROOT's package manager prototype
+
+root-get is the project centric package manager for ROOT. The application provides simplified build and install process for ROOT packages and modules. 
+
+## Getting started
+
+## Manifest file
+The installation procedure in root-get has manifest files at its base. Manifest file is structured as yml file and provides the necessary attributes regarding an entity. (hereafter entity refers to package or module)
+Package manifest contains the package attributes along with specifications regarding its modules.  
+Module manifest contains attributes for a specific module.
+The necessary attributes in any manifest file are :
+- name (module/package)
+- packageurl (In case of downloading the entity and using it)
+- path (The absolute path for the entity if present in system)
+- deps (Dependencies for the entity)
+- publicheaders (Headers for the entity)
+- sources 
+- targets 
+
+## Environment variables
+Setting environment variables before using root-get is important to allow root-get to search for various required files from the system.
+Important environment variables :
+- ROOTSOURCES
+
+The rootbase source files directory. Useful in build and install of root packages and modules.
+
+- ROOT_PKG_PATH
+
+The directory path where user can get the installed package/module.
+
+- ROOTSYS
```

Comment:
Not exactly, ROOTSYS is also the directory where you unpacked the ROOT (e.g. ROOT  base)

Reply:
Here, Line 31 is the explanation for ROOT_PKG_PATH, for ROOTSYS the explanation is below it.